src/core/BundleSplit.ts

```diff
@@ -42,6 +42,7 @@ export class BundleSplit {
         config.plugins = [].concat(config.plugins || [])
         config.standalone = false;
         const fuse = FuseBox.init(config);
+        new Bundle(name, fuse, producer);
```

Comment:
If an instance of bundle isn't created for the split, it'll error out wherever this instance of fuse tries to access bundle props like `bundle.name` for example.